types/mongoose/index.d.ts

```diff
@@ -26,44 +26,7 @@
  * - Check the files off as you go. Some files below might not have anything in them. That's ok, this
  *   is just a simple heuristic to keep track of our progress.
  *
- * TODO for version 4.x [updated][tested]:
- * [x][x] index.js
- * [x][x] querystream.js
- * [x][x] connection.js
- * [x][x] utils.js
- * [x][x] browser.js
- * [x][x] drivers/node-mongodb-native/collection.js
- * [x][x] drivers/node-mongodb-native/connection.js
- * [x][x] error/messages.js
- * [x][x] error/validation.js
- * [x][x] error.js
- * [x][x] querycursor.js
- * [x][x] virtualtype.js
- * [x][x] schema.js
- * [x][x] document.js
- * [x][x] types/subdocument.js
- * [x][x] types/array.js
- * [x][x] types/documentarray.js
- * [x][x] types/buffer.js
- * [x][x] types/objectid.js
- * [x][x] types/embedded.js
- * [x][x] query.js
- * [x][x] schema/array.js
- * [x][x] schema/string.js
- * [x][x] schema/documentarray.js
- * [x][x] schema/number.js
- * [x][x] schema/date.js
- * [x][x] schema/buffer.js
- * [x][x] schema/boolean.js
- * [x][x] schema/objectid.js
- * [x][x] schema/mixed.js
- * [x][x] schema/embedded.js
- * [x][x] aggregate.js
- * [x][x] schematype.js
- * [x][x] promise.js
- * [x][x] ES6Promise.js
- * [x][x] model.js
- * [x][x] collection.js
+ * TODO for version 5.x [updated][tested]:
```

Comment:
I think you can remove this line too. It was from a long time ago and now I don't think we really need it anymore.